packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -377,6 +411,25 @@ class OpPerfTelemetry {
 			this.clientSequenceNumberForLatencyStatistics = undefined;
 			this.latencyStatistics.delete(message.clientSequenceNumber);
 		}
+
+		if (isRuntimeMessage(message)) {
+			// Sampled logging of Ops that have been processed by the current client, the NoOp sent and the
+			// size of the ops processed within one sampling window of this log event.
+			// This data will be used to monitor the efficiency of NoOp-heuristics or to get approximate collab window size.
+			this.opsLogger.sendPerformanceEvent({
+				eventName: "OpStats",
+				// Logging as 'details' property to avoid adding new column name to the log tables */
+				details: {
+					// Count of the ops processed by the current client. Note: these counts are after
+					// compression/grouping/chunking (if enabled) of the ops.
+					processedOpCount: OpPerfTelemetry.PROCESSED_OPS_SAMPLE_RATE,
```

Comment:
I know this is merged but ran into this event while looking at telemetry with Fluid Devtools and was very confused to see processedOpCount=500 all the time as I saw new ops be processed, because both the property name and the comment suggest it should be a changing number. @pragya91 is there some pending work here, maybe?

Reply:
There is no pending work. The event is designed so that it is sampled to log after every 500 ops that are processed, along with the no-ops count seen in that window. I can update the code comment to make this clearer. Let me know if you have recommendations on updating the prop name. Happy to change it.